### PR TITLE
 code of categories component #4

### PR DESCRIPTION
**Pull request by kholoudfann**

### CODE REVIEW

**Review by bayan-404 — APPROVED** [review]

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,10 @@
 import React from 'react';
+// import Category from './category'
```
Comment (yaqootturman) [inline src/App.js]: Delete this comment

File: src/component/category.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from "react";
+
+  const Category = () => {
```
Comment (yaqootturman) [inline src/component/category.js]: Change the name of the component to Capital at first
and make your code cleaner and more readable --the indentation as specific

File: src/component/category.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+     <input type='radio' value='Programming' name='category'/> 
+     Programming
+     <input type='radio' value='Any' name='category'/> 
+    Any
+    <input type='radio' value='Miscellaneous' name='category'/> 
+    Miscelleneous
+    <input type='radio' value='Dark' name='category'/> 
+    Dark
+      </div> 
+<p>{Joke}</p>
```
Comment (yaqootturman) [inline src/component/category.js]: You created the component in the same branch so you can call it here

**Review by yaqootturman — CHANGES_REQUESTED** [review]

**Review by Rahaf-96 — APPROVED** [review]

**Review by yaqootturman — APPROVED** [review]

**Review by Hussein-shahatet97 — APPROVED** [review]
:100:

**Pull request merged**